Eukalyptus-Dreizack/deepcity-framework
# Fix package names

## Description
Fixed the package names to be prefixed with @eukalyptus-dreizack/ to deploy the framework to our npmjs scope..

## Related issue
#70 

## How can this be tested?
- Check the package name of all package.json files to ensure that they are prefixed with @eukalyptus-dreizack/
- Check if the deployment of our packages to npmjs was successfull.

## Motivation and context
The packages should be deployed to a scope where all team members hve access to.
